shared/js/background/devbuild.js

```diff
@@ -37,7 +40,8 @@ export default function initDebugBuild () {
         setListContents,
         getListContents,
         companies: Companies,
-        ntts: createNewtabTrackerStatsDebugApi()
+        ntts: createNewtabTrackerStatsDebugApi(),
+        sendPageloadsWithAdAttributionPixelAndResetCount
```

Comment:
Was this just for debugging?

Reply:
No, I exposed `sendPageloadsWithAdAttributionPixelAndResetCount` here so I could call it from the integration tests.